.github/CODEOWNERS

```diff
@@ -41,6 +41,7 @@
 airflow-core/src/airflow/ui/public/i18n/locales/ar/ @shahar1 @hussein-awala # + @ahmadtfarhan
 airflow-core/src/airflow/ui/public/i18n/locales/ca/ @jscheffl @bugraoz93 # + @ecodina
 airflow-core/src/airflow/ui/public/i18n/locales/de/ @jscheffl # + @TJaniF @m1racoli
+airflow-core/src/airflow/ui/public/i18n/locales/el/ @PApostol
```

Comment:
Only committers can be added to codeowners file, you probably need to find a translation sponsor which is a committer which can help in merging and add yourself as comment being the engaged translator.

Reply:
I'll volunteer for that. Put me down @PApostol